src/swap/index.js

```diff
@@ -0,0 +1,82 @@
+    signature: Buffer.from([0x30, 0x44, 0x2, 0x20, 0x6A, 0x57, 0xC9, 0xED, 0x40, 0x30, 0xCB, 0x21, 0x70, 0xA3, 0x45, 0x74, 0x73, 0x75, 0x7E, 0x6, 0x3F, 0xE7, 0x27, 0xC1, 0x43, 0x0, 0x2, 0x51, 0xD9, 0xE9, 0x7F, 0x27, 0x89, 0x90, 0x29, 0x80, 0x2, 0x20, 0x7B, 0x11, 0x97, 0xDA, 0x75, 0xFE, 0x90, 0x9C, 0xB9, 0xE2, 0x85, 0x77, 0x5A, 0xF9, 0xC9, 0x6A, 0x62, 0x69, 0x5C, 0x18, 0x74, 0x56, 0x5B, 0xC6, 0x7B, 0x18, 0xF9, 0xF7, 0xC6, 0x11, 0xDE, 0x82]) // prettier-ignore
+  },
+};
+
+// Fixme These configuration/signature pairs will tell the swap app which currency app to open and sign with
+// alongside which parameters (derivation path and so on). We should be able to generate this instead
+const swapCurrencyConfigs: { [string]: SwapCurrencyNameAndSignature } = {
+  bitcoin: {
+    config: Buffer.from([0x3, 0x42, 0x54, 0x43, 0x7, 0x42, 0x69, 0x74, 0x63, 0x6F, 0x69, 0x6E, 0x0]), // prettier-ignore
+    signature: Buffer.from([0x30, 0x45, 0x2, 0x21, 0x0, 0x97, 0xCE, 0x38, 0xC5, 0x15, 0x60, 0xFD, 0xBF, 0x2D, 0xBE, 0x87, 0x3C, 0x22, 0xFC, 0xBE, 0xC0, 0x7B, 0x58, 0xF1, 0x33, 0x13, 0x24, 0x5E, 0x8F, 0xB6, 0x4F, 0xDB, 0x2B, 0x32, 0xD4, 0x55, 0xE0, 0x2, 0x20, 0xA, 0xE6, 0x3C, 0xDA, 0x3C, 0xD3, 0xCC, 0x3, 0x43, 0x65, 0x3B, 0x38, 0xE5, 0xC2, 0xC5, 0xC2, 0x7B, 0xFE, 0x47, 0xB3, 0x5D, 0x25, 0x30, 0x26, 0x37, 0x89, 0x63, 0x47, 0x59, 0xC6, 0xD5, 0xB4]) // prettier-ignore
+  },
+  litecoin: {
+    config: Buffer.from([0x3, 0x4C, 0x54, 0x43, 0x8, 0x4C, 0x69, 0x74, 0x65, 0x63, 0x6F, 0x69, 0x6E, 0x0]), // prettier-ignore
+    signature: Buffer.from([0x30, 0x44, 0x2, 0x20, 0x2A, 0xD5, 0x61, 0xDD, 0x8B, 0x3B, 0x98, 0x9C, 0xEA, 0x41, 0x1D, 0x4C, 0x1D, 0x8A, 0x6A, 0x62, 0x71, 0xB4, 0xF7, 0x2F, 0xB1, 0x66, 0x25, 0x9C, 0x89, 0x8F, 0xFC, 0x11, 0x70, 0x74, 0x72, 0xEB, 0x2, 0x20, 0x34, 0x6A, 0xA8, 0x95, 0x5E, 0x8E, 0x6B, 0xE6, 0x5D, 0x2F, 0xC6, 0x58, 0x36, 0x3A, 0xF4, 0x31, 0xD7, 0x1E, 0xA8, 0x65, 0xCC, 0xC8, 0x3D, 0xD4, 0xA7, 0x14, 0x9B, 0xEF, 0xF9, 0x35, 0x21, 0xB8]) // prettier-ignore
+  },
+  ethereum: {
+    config: Buffer.from([0x3, 0x45, 0x54, 0x48, 0x8, 0x45, 0x74, 0x68, 0x65, 0x72, 0x65, 0x75, 0x6D, 0x5, 0x3, 0x45, 0x54, 0x48, 0x12]), // prettier-ignore
+    signature: Buffer.from([0x30, 0x44, 0x2, 0x20, 0x5F, 0xEF, 0x55, 0x54, 0x36, 0xAC, 0xE6, 0x76, 0x8B, 0xA6, 0x66, 0x6A, 0xB7, 0x83, 0x4B, 0x16, 0x87, 0x36, 0x9B, 0x6, 0x27, 0x47, 0x78, 0xFD, 0xD2, 0x8E, 0xB, 0x88, 0xD3, 0x43, 0x4D, 0xE9, 0x2, 0x20, 0x6A, 0x67, 0xF4, 0xCF, 0x62, 0x2C, 0x6B, 0xBC, 0x12, 0x7B, 0xBE, 0x86, 0x6A, 0x7B, 0x65, 0x40, 0xBD, 0x97, 0xE9, 0x1A, 0xCB, 0x38, 0xF1, 0x79, 0xA5, 0x54, 0xE6, 0x94, 0xBD, 0xEA, 0x50, 0x43]) // prettier-ignore
+  },
+};
```

Comment:
Can these be moved directly in the cryptocurrencies.js 🤔

I would also prefer to just use an hexa string so we can later externalize as a json